lib/constants.dart

```diff
@@ -256,3 +256,18 @@ String soundMeterError = 'Sound sensor error:';
 String soundMeterInitialError = 'Sound sensor initialization error:';
 String db = 'dB';
 String soundMeterTitle = 'Sound Meter';
+String minLabel = 'Min';
+String maxLabel = 'Max';
+String avgLabel = 'Avg';
+String instrumentMenu = 'Instruments';
+String loggedDataMenu = 'Logged Data';
+String connectDeviceMenu = 'Connect Device';
+String configFileMenu = 'Generate Config File';
+String settingsMenu = 'Settings';
+String aboutUsMenu = 'About Us';
```

Comment:
I noticed that there are few duplicates like 'About Us' here and also at the beginning. Could you please check for those duplicates and fix them?